src/components/WishList/WishList.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import classes from './WishList.module.scss';
+import WishListItem from './WishListItem/WishListItem';
+import './WishList.module.scss';
+
+interface WishTableProps {
+  rows: Row[];
+}
+
+interface Row {
+  id: string;
+  name: string;
+  img: string;
+  price: number;
+}
+
+const WishList: React.FC<WishTableProps> = ({ rows }) => {
```

Comment:
what if we change layout to grid?
```suggestion
const WishList: React.FC<WishTableProps> = ({ rows }) => {
```